geode-core/src/main/java/org/apache/geode/internal/cache/EntryExpiryTask.java

```diff
@@ -326,10 +326,9 @@ protected void basicPerformTimeout(boolean isPending) throws CacheException {
         }
         if (hasExpired(getNow(), expTime)) {
           if (logger.isTraceEnabled()) {
-            // NOTE: original finer message used this.toString() twice
             logger.trace(
-                "{}.performTimeout().getExpirationTime() is {}; {}.expire({}). ttlExpiration: {}, idleExpiration: {}, ttlAttrs: {}, idleAttrs: {} action is: {}",
-                this, expTime, this, action, ttl, idle, getTTLAttributes(), getIdleAttributes());
+                "{}.performTimeout().getExpirationTime() is {}; ttlExpiration:{}, idleExpiration:{}, ttlAttrs:{}, idleAttrs:{} action is:{}",
```

Comment:
`this` is passed as parameter for `{}.performTimeout().getExpirationTime()`. It will call `EntryExpiryTask.toString()` which returns a long string description with spaces. It is not easy to read. Perhaps passing `this.getClass().getSimpleName()` would be better. And the method name should be `basicPerformTimeout` instead of `performTimeout()`.